From d1954ab4c9aa67a0d5a2457120cbc8c02594de6a Mon Sep 17 00:00:00 2001
From: Jeff Layton <jlayton@redhat.com>
Date: Mon, 30 Oct 2017 11:11:56 -0400
Subject: [PATCH] f2fs: don't bother with inode->i_version
Git-commit: d1954ab4c9aa67a0d5a2457120cbc8c02594de6a
Patch-mainline: v4.15-rc1
References: bsc#1081186

f2fs does not set the SB_I_VERSION flag, so the i_version will never
be incremented on write. It was recently changed to increment the
i_version on a quota write, which isn't necessary here.

Signed-off-by: Jeff Layton <jlayton@redhat.com>
Reviewed-by: Chao Yu <yuchao0@huawei.com>
Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>
Acked-by: Jan Kara <jack@suse.cz>

---
 fs/f2fs/super.c |    1 -
 1 file changed, 1 deletion(-)

--- a/fs/f2fs/super.c
+++ b/fs/f2fs/super.c
@@ -613,7 +613,6 @@ static struct inode *f2fs_alloc_inode(st
 	init_once((void *) fi);
 
 	/* Initialize f2fs-specific inode info */
-	fi->vfs_inode.i_version = 1;
 	atomic_set(&fi->dirty_pages, 0);
 	fi->i_current_depth = 1;
 	fi->i_advise = 0;
